lib/game.rb

```diff
@@ -5,5 +5,22 @@ def intro
     puts "#{welcome}\n#{play}" 
     return "#{welcome}\n#{play}" 
   end
+
```

Comment:
Great way to receive user input from our player. The loop will work to serve our purposes.